starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,13 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  @Test
+  public void create(){
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.inventoryDAO.create(inventory);
+    Assert.assertEquals(1,this.inventoryDAO.findAll().size());
```

Comment:
I would prefer to use the mongoTemplate.findAll() method for checking validity. That way your tests aren't reliant on the implementation of inventoryDAO.findAll()

Reply:
Changed to mongoTemplate.findAll().